### PR TITLE
chore: add `stats` as a pull request header type and scope

### PR DESCRIPTION
**Pull request by kwvg**
## Additional Information

Would allow tagging https://github.com/dashpay/dash/pull/5167, https://github.com/dashpay/dash/pull/6267 and https://github.com/dashpay/dash/pull/6237 as `feat(stats)`.

## Breaking Changes

None.

## Checklist:

- [x] I have performed a self-review of my own code **(note: N/A)**
- [x] I have commented my code, particularly in hard-to-understand areas **(note: N/A)**
- [x] I have added or updated relevant unit/integration/functional/e2e tests **(note: N/A)**
- [x] I have made corresponding changes to the documentation **(note: N/A)**
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_ **(note: N/A)**

### CODE REVIEW

**Comment by thephez** [comment]
> ## Additional Information
> Would allow tagging [dash#5167](https://github.com/dashpay/dash/pull/5167) and its successor PR as `feat(stats)` and [dash#6237](https://github.com/dashpay/dash/pull/6237) as `stats`.
> 

I think you'll be fine since we've changed the default branch to develop. But if it doesn't work as expected it, check for where it actually runs from.

**Review by knst — APPROVED** [review]
LGTM a6f7339b9d3b842ba6fb7241c39e50bd8ab1adb2

File: .github/workflows/semantic-pull-request.yml [hunk 1]
```diff
@@ -47,3 +48,4 @@ jobs:
             utils
             wallet
             zmq
+            stats
```
Comment (UdjinM6) [inline .github/workflows/semantic-pull-request.yml:51]: should update https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#creating-the-pull-request

File: .github/workflows/semantic-pull-request.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ jobs:
             revert
             trivial
             backport
+            stats
```
Comment (UdjinM6) [inline .github/workflows/semantic-pull-request.yml:37]: `stats` type doesn't sound right to me tbh, #6237 should be `feat(stats)` imo
```suggestion
```

**Review by PastaPastaPasta — APPROVED** [review]
utACK 4faf35f7491f8213a36d1652010bcc2bdff94c1d

**Review by UdjinM6 — APPROVED** [review]
utACK 4faf35f7491f8213a36d1652010bcc2bdff94c1d

**Review by thephez — APPROVED** [review]
LGTM :+1: 
utACK 4faf35f7491f8213a36d1652010bcc2bdff94c1d

**Pull request merged**